src/extension/background-script/actions/ln/transactions.ts

```diff
@@ -0,0 +1,46 @@
+import utils from "~/common/lib/utils";
+import { ConnectorTransaction } from "~/extension/background-script/connectors/connector.interface";
+import state from "~/extension/background-script/state";
+import type { MessageTransactions } from "~/types";
+
+const transactions = async (message: MessageTransactions) => {
```

Comment:
should this be `getTransactions`?

Reply:
Also rename the message type: MessageGetTransactions

Reply:
resolved